### ISSUE TITLE
onPress of Touch trigger twice when on moblie phone

### ISSUE THREAD

**Issue by YvetteLau** [necolas/react-native-web#1571]
I'm so sorry for not describing it clearly last time.

It works well on PC, but will trigger twice on moblie phone or safari of ios Simulator (always).

Alow if I turns on phone mode in browser ,it sometime trigger twice. 

I have checked my react and react-dom version.

My code like this:

```js
<TouchableHighlight onPress={()=>{
    alert('You pressed me')
 }}>
     <View style={styles.touchStyle}>
         <Text >
            press me
        </Text>
     </View>
</TouchableHighlight>
```
when I click the alertDialog, onPress will trigger one more.

It's sees that when I click the alert dialog, touchableHandleResponderGrant of Touchable will trigger.

>My Version is :
- react-native:0.59.9
- react:16.10.2
- react-dom:16.10.2

**Comment by YvetteLau** [comment]
> More information:

- I test it on iphone, it just trigger once.
- work on safari of ios Simulator, always trigger twice.
- But on andorid phone, it sometime trigger twice.
- if I turns on phone mode in browser (google), it sometime trigger twice.

I don't know how to handle it, hope your help.

Thanks very much.

**Comment by necolas** [comment]
I still don't understand. Please can you share a video?

**Comment by YvetteLau** [comment]
> I still don't understand. Please can you share a video?

- In ios simulator

![simulator](https://user-images.githubusercontent.com/20203157/78096374-f5788500-740b-11ea-9dbe-5188f4fd31bd.gif)

- chrome phone mode

![chrome-phone-mode](https://user-images.githubusercontent.com/20203157/78096445-25278d00-740c-11ea-97ca-4b8e3a2eecec.gif)

**Comment by YvetteLau** [comment]
> I still don't understand. Please can you share a video?

My test code: https://github.com/YvetteLau/react-native-web-demo.git

Thanks very much.

**Comment by YvetteLau** [comment]
When I click 'close'/'ok'  button of the alert-dialog 

touchableHandleResponderGrant of Touchable will be called again sometimes.

**Comment by necolas** [comment]
Thanks. I can reproduce the issue in this example https://codesandbox.io/embed/elastic-swirles-x8jn5?codemirror=1

This happens because browsers fire `mousedown` after `touchstart`, and RNW ignores mouse events within 1 second of touch start https://github.com/necolas/react-native-web/blob/master/packages/react-native-web/src/modules/ResponderEventPlugin/index.js#L70

When an alert

Ref #1570

**Comment by YvetteLau** [comment]
> Thanks. I can reproduce the issue in this example https://codesandbox.io/embed/elastic-swirles-x8jn5?codemirror=1
> 
> This happens because browsers fire `mousedown` after `touchstart`, and RNW ignores mouse events within 1 second of touch start https://github.com/necolas/react-native-web/blob/master/packages/react-native-web/src/modules/ResponderEventPlugin/index.js#L70
> 
> So if you try to close the alert as soon as it opens (within 1 second), no extra alert opens. When a native alert dialog opens it delays the emulated mouse events for as long as it is open.
> 
> #1568 is changing how the responder system is implemented, and I think it will avoid this problem too, because it doesn't rely on a time threshold.
> 
> Ref #1570

I see. Thanks very much.

**Comment by necolas** [comment]
I can confirm that this issue is fixed in the `0.0.0-3ada692a3` canary release mentioned in #1538

**Comment by YvetteLau** [comment]
> I can confirm that this issue is fixed in the `0.0.0-3ada692a3` canary release mentioned in #1538

That's great, thanks again.
